tox.ini

```diff
@@ -410,7 +410,7 @@ usedevelop = True
 deps =
     grpcio==1.43.0
     py-eth-sig-utils==0.4.0
-    numpy<1.22,>=1.21.4
+    numpy==1.22.0
     optuna<2.11,>=2.10.0
     pandas<1.4,>=1.3.4
     pandas-stubs<1.2.1,>=1.2.0.39
```

Comment:
Noticed that I forgot to pin these here in #1220.
Also, not sure if they are needed for `autonomy analyse abci docstrings`?